Library/ViewModels/LoginToutViewModelTests.swift

```diff
@@ -31,6 +31,8 @@ final class LoginToutViewModelTests: TestCase {
   override func setUp() {
     super.setUp()
 
+    self.vm = LoginToutViewModel()
```

Comment:
In order for the test to pass we needed to create the VM after the scheduler is set in the environment. We've had to do this in a number of places.